src/main/scala/com.gu.gibbons/services/interpreters/BonoboInterpreter.scala

```diff
@@ -30,21 +32,28 @@ class BonoboInterpreter(config: Settings,
     for {
       _ <- logger.info(s"Getting all the users created $jadis ago")
       millis = jadis.toEpochMilli
-      users <- getUsersMatching(
-        not(attributeExists('remindedAt)) and ('extendedAt <= millis or (not(attributeExists('extendedAt)) and 'createdAt <= millis))
-      )
+      users <- getItems(usersTable,
+                        not(attributeExists('remindedAt)) and ('extendedAt <= millis or (not(
+                          attributeExists('extendedAt)
+                        ) and 'createdAt <= millis)))
```

Comment:
This condition is hard to read. Would you mind explaining it in plain English?

Reply:
It is a translation of the logical assertion _¬p ∧ (q ∧ r ∨ ¬q ∧ s)_ where `q ∧ r` can be simplified to just `r` because `r ⇒ q`.